extensions/elasticsearch-rest-client-common/runtime/src/main/java/io/quarkus/elasticsearch/restclient/common/runtime/graal/Substitute_RestClient.java

```diff
@@ -38,7 +38,7 @@
 final class Substitute_RestClient {
 
     @Alias
-    private ConcurrentMap<HttpHost, DeadHostState> blacklist;
+    private ConcurrentMap<HttpHost, DeadHostState> denylist;
```

Comment:
IIRC, the `Alias` has to exactly have the name of the field that is being referenced. In this case it has to match the: https://github.com/elastic/elasticsearch/blob/master/client/rest/src/main/java/org/elasticsearch/client/RestClient.java#L115 field

Reply:
Yes, so we can't apply this change without a change in the Elastic code source.
@paulrobinson can you revert this one?

Reply:
Reverted.